### PR TITLE
Milestone: Make list interactive

### PR DESCRIPTION
**Pull request by Timbar09**
**Changes**

- Added a new JavaScript file and imported it as a module
- Implemented a function that updates the task complete key's value to true or false
- Implemented a function for the "Clear all completed" button

### CODE REVIEW

File: src/modules/taskStatus.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable arrow-body-style */
+
+import { storeTask, render } from './functions.js';
```
Comment (KDlamini) [inline src/modules/taskStatus.js]: Please try to find a way to solve the linter errors than disabling them. They are meant to teach you how to use the best practices when writing code no matter how illogical they may seem sometimes.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Timbar09 🙋‍♀️,  

Excellent work! 💯 
You have done a great job making the To-do list interactive. 🎊 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Very beautiful UI and use of color for background highlights
✔️  Task object's values for `completed` is updated upon user actions
✔️  A function for the "Clear all completed" button uses the `Array.filter()` method
✔️  Well done updating the `index` values after clearing completed tasks 👏🏾💯 
✔️  No linter errors

## Required Changes ♻️
 
 - [ ] Please make sure that the README does not have broken links. Since you have not deployed the project yet, kindly leave a `coming soon...` text rather than a broken link.
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/taskStatus.js (hunk 1 above)
Comment (Timbar09) [reply src/modules/taskStatus.js]: I got you @KDlamini

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Readme Updated  ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  As mentioned by the previous reviewer kindly remove the eslint disabling commands and try to fix the issue 🙏 _**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Timbar09** [comment]
Thanks, @ichala.

**Comment by ichala** [comment]
> @ichala is this really necessary? I disabled just 1 rule.

Well it was requested by the previous reviewer, and it was a bit easy to fix so i suggest fixing it 🙏 if you need help let me know

**Comment by Timbar09** [comment]
for

> > @ichala is this really necessary? I disabled just 1 rule.
> 
> Well it was requested by the previous reviewer, and it was a bit easy to fix so I suggest fixing it 🙏 if you need help let me know

Forget what I just said. 

Your review has helped me immensely with the issues I was having with linters. I configured the liners correctly and all is well.

Thanks again. 🙏

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Timbar09 , 

To highlight 👍🏽 
✔️ No disabled linters
✔️ No linter errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**